docs/source/developer/freeradius.rst

```diff
@@ -110,7 +110,7 @@ Example configuration using the PostgreSQL database:
 Configure the SQL counters
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The ``sqlcounter`` module is used to `enforce session limits <./enforcing_limits.html>`_.
+The ``sqlcounter`` module is used to `enforce session limits </user/enforcing_limits.html>`_.
```

Comment:
I found links starting with `./` do not throw errors during build even if the file doesn't exist. 
Hence, I converted all of them to `/<path>/`